### PR TITLE
Add enhancedMeasurement to the feature flags.

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #7470 

## Relevant technical choices

Implemented per IB

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: feature-flags.json [hunk 1]
```diff
@@ -4,5 +4,6 @@
 	"dashboardSharing",
 	"ga4Reporting",
 	"gm3Components",
-	"userInput"
+	"userInput",
+	"enhancedMeasurement"
```
Comment (hussain-t) [inline feature-flags.json:8]: Could we move it just below `dashboardSharing` to arrange it in alphabetical order?

**Review by hussain-t — CHANGES_REQUESTED** [review]
Thanka, @zutigrm. This looks good. However, I have left a nitpicky comment. Please take a look.

File: feature-flags.json (hunk 1 above)
Comment (zutigrm) [reply feature-flags.json]: Order changed. Thanks. I thought latest flags go as last index in array.

**Review by hussain-t — APPROVED** [review]
Thanka, @zutigrm. LGTM 👍

**Review by tofumatt — APPROVED** [review]

**Pull request merged**